src/components/mega-nav/components/MegaNavStaticLink.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <li
+        class="vs-mega-nav-static-link"
+        :class="isFullWidth ? 'vs-mega-nav-static-link--full-width' : ''"
+        data-test="vs-mega-nav-static-link"
+        role="presentation"
+    >
+        <div>
```

Comment:
Do we need this div inside the list item? It adds an unnecessary extra element for the browser to paint

Reply:
No I don't think so, it was a remnant of an earlier style set up for them but it seems the same without it